### ISSUE TITLE
Unable to get a corpus with all English-language annotated documents

### ISSUE THREAD

**Issue by SUN-Wenjun** [ManifestoProject/manifestoR#5]
I tried to follow the tutorial to get a corpus with all English-language annotated documents by 
`english_annotated <- mp_availability(TRUE) %>% filter(annotations==TRUE & language == "english")
mp_corpus(english_annotated)`

But it sends out an error,'Error in is.naorstringna(manifesto_id) : cannot found object 'manifesto_id''

**Comment by SUN-Wenjun** [comment]
okay I think it's because `mp_corpus` requires a manifesto_id column but the input english_annotated data frame does not have one. I got away by adding `%>% mutate(manifesto_id=paste0(party, "_", date))` after the filter()

**Comment by mirkobunse** [comment]
In the beginning of 2019, I wrote a script which successfully obtained all annotated documents. It looks just like your code:

```R
annotation_availability <- mp_availability(TRUE) %>% filter(annotations == TRUE)
annotated_corpus <- mp_corpus(annotation_availability)
```

This script is now giving me an ```ERROR: Can't combine `language` <logical> and `language` <character>.``` I solved the issue by replacing `mp_availability(TRUE)` in the script above with `mp_metadata(TRUE)`.

**Comment by tburst** [comment]
The problem was likely caused by an incompatibility with newer dplyr versions (>= 0.8.0) and should be fixed with the ManifestoR 1.5 update.
